### ISSUE TITLE
train ConvolutionModel  model,it seems failed

### ISSUE THREAD

**Issue by snowlord** [codekansas/keras-language-modeling#27]
when i train CNN model,it epochs 2 twice ,the loss value became nan. it seems getting in local minmize.
![image](https://cloud.githubusercontent.com/assets/8564700/19342657/48cc11d2-9165-11e6-9abe-320f24da3541.png)

**Comment by metalaman** [comment]
I've ported the CNN-LSTM model implemented here, on [PyTorch](https://github.com/metalaman/insuranceqa-pytorch) and my loss also becomes 0.0 for the second, third and fourth epochs and then rises again and keeps oscillating. Is it a Learning Rate issue.
